### PR TITLE
if you are shifting you go backwards in left click now

### PR DESCRIPTION
**Pull request by TheCricket**

### CODE REVIEW

**Comment by minecreatr** [comment]
I say yes, @tterrag1098 Your say?

File: src/main/java/team/chisel/item/chisel/ChiselController.java [hunk 1]
```diff
@@ -44,6 +44,9 @@ public void onInteract(PlayerInteractEvent event) {
 
 		switch (event.action) {
 		case LEFT_CLICK_BLOCK:
+			boolean shifting = false;
```
Comment (tterrag1098) [inline src/main/java/team/chisel/item/chisel/ChiselController.java]: Honestly this bit is redundant, just check `isSneaking()` below.

Comment (TheCricket) [inline src/main/java/team/chisel/item/chisel/ChiselController.java]: Yes but I made it so in case you need to use it somewhere else later on you have it

Comment (tterrag1098) [inline src/main/java/team/chisel/item/chisel/ChiselController.java]: Even then, it should just be `boolean shifting = event.player.isSneaking();`

Comment (TheCricket) [inline src/main/java/team/chisel/item/chisel/ChiselController.java]: Oh yeah sorry need caffine :P

**Pull request merged**